asylo/third_party/fortanix/README.md

```diff
@@ -0,0 +1,4 @@
+This folder contains the source code of the Rust crate `asylo`
```

Comment:
Since this repository is the primary source for this crate, is the `third_party` directory an appropriate location?

Reply:
moved to `asylo/rust-asylo`